lib/crummy/version.rb

```diff
@@ -1,6 +1,6 @@
 module Crummy
   MAJOR = 1
```

Comment:
I see the breaking change (removing ruby 1.9 support) - in that case, this should be:
```suggestion
  MAJOR = 2
  MINOR = 0
  PATCH = 0
```